### ISSUE TITLE
Save as PDF throws Javascript exception

### ISSUE THREAD

**Issue by Venkat5694** [hql287/Manta#110]
Please answer these question so I can help you faster.

**Is it a bug or a feature request?**

- [x] It's a bug.
- [ ] I'm requesting a new feature.

**Which version are you using?**

- [x] macOS
- [ ] Windows
- [ ] Linux

**Which is the version/distribution of the OS?**

 I'm using macOS Sierra 10.12.6

**Describe your issue**

When trying to save a Invoice a JS exception is thrown.

```
Uncaught Exception:
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.join (path.js:1211:7)
    at EventEmitter.ipcMain.on (/Users/rvenkataraman/Documents/work/Manta/main/save-pdf.js:9:24)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:194:7)
    at WebContents.<anonymous> (/Users/rvenkataraman/Documents/work/Manta/node_modules/electron/dist/Electron.app/Contents/Resources/electron.asar/browser/api/web-contents.js:247:13)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:194:7)
```

**Steps to replicate**

Eg: 
1. Run the App from CLI using yarn
2. Create an invoice and try saving it as PDF.
3. It throws a JS exception
4. Go to settings and change the PDF folder
5. Go back and open the invoice and save as PDF.
6. Same exception is thrown again.

**Comment by hql287** [comment]
Hm, looks like a duplicate of https://github.com/hql287/Manta/issues/112 ?

**Comment by hql287** [comment]
OK, #109 introduced some changes in the way I store export path but I forgot to update it with the saveAsPDF script. Should be fixed in #114.
Thanks for reporting this 👍
